.travis.yml

```diff
@@ -1,7 +1,15 @@
 language: python
-python:
-   - "2.7"
-   - "3.4"
+jobs:
+  include:
+    - name: "Python 2.7 on Xenial"
+      python: "2.7"
+      dist: "xenial"
+    - name: "Python 3.5 on Xenial"
+      python: "3.5"
+      dist: "xenial"
+    - name: "Python 3.6 on Bionic"
+      python: "3.6"
+      dist: "bionic"
```

Comment:
What about adding tests for `focal` (20.04) with `python` 3.8?

Reply:
Tests are failing on focal :)

Reply:
It seems that bug either in coverage.py. Maybe something was broken in Py3.8 itself, didn't investigate that as we don't plan to migrate on focal soon.